Material.Blazor/Material.Blazor.csproj

```diff
@@ -1,108 +1,117 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
 
-  <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <RazorLangVersion>3.0</RazorLangVersion>
-    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-    <VersionPrefix>1.0.0</VersionPrefix>
-    <VersionSuffix></VersionSuffix>
-    <PackageLicenseFile></PackageLicenseFile>
-    <Authors>Simon Ziegler of Dioptra and Mark Stega of Optimium Health</Authors>
-    <Company>Material.Blazor</Company>
-    <Description>A lightweight Material Theme component library for Blazor. Intended to adhere rigorously to Material Theme guidelines in https://material.io. API and other documentation can be found at https://material-blazor.com/docs.</Description>
-    <RepositoryUrl>https://github.com/Material-Blazor/Material.Blazor</RepositoryUrl>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <PackageReleaseNotes></PackageReleaseNotes>
-    <NeutralLanguage>en</NeutralLanguage>
-    <Copyright>Dioptra and Optimium Health</Copyright>
-    <PackageProjectUrl>https://material-blazor.com</PackageProjectUrl>
-    <Product>Material.Blazor</Product>
-    <UserSecretsId>d7509b6c-bc5c-4a56-a750-09c8ae4aeb85</UserSecretsId>
-  </PropertyGroup>
+	<PropertyGroup>
```

Comment:
Apologies for formatting the entire document. Actual changes only further down...